src/pages/sunhoLee/Main/Fedd/Comment/Comment.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import CommentList from './CommentList';
+
+let id = 4;
+
+class Comment extends Component {
+  constructor() {
+    super();
+    this.state = {
+      commentView: [],
+      comments: '',
+    };
+  }
+
+  componentDidMount() {
+    this.setState({
+      commentView: this.props.comments,
+    });
+  }
+
+  getCommentValue = e => {
+    this.setState({
+      comments: e.target.value,
+    });
+  };
+
+  uploadComment = e => {
+    if (this.state.comments.length > 0) {
+      this.state.commentView.push({
+        id: id,
+        userId: 'dltjsgho',
+        content: this.state.comments,
+      });
```

Comment:
- React의 state값은 불변하게 다뤄야 합니다(react에서 state의 변경 여부를 판단할 때의 동작과 연관되어있습니다)
- Array.push, Array.unshift 등의 원본 배열을 변경하는 메소드나, 객체의 속성값을 직접 변경하는 경우는 객체나 배열을 가변하게 사용한 것이기 때문에 좋지 않습니다.
- map, filter, concat 등 새로운 배열을 만들어내는 함수를 통해서 새로운 배열을 만들고 그 배열을 새로운 state에 할당하는 식으로 진행해주세요
```suggestion
      const newCommentView = this.state.commentView.concat({
        id: id,
        userId: 'dltjsgho',
        content: this.state.comments,
      });
      
      this.setState({commentView:newCommentView, comments: ""})
```